src/lib/checkUser.js

```diff
@@ -0,0 +1,6 @@
+import { redirect } from "next/navigation";
```

Comment:
also need a catch for the '/login' that if they are already logged in, it redirects back to '/' but we also don't have a logout button yet

Reply:
i have commented out code that tried to do this in the login route, but i dropped it to move on since middleware should probably solve this. i got stuck on next13 client side redirects. it suggests i use middleware and next.config